src/parquet/column/properties.h

```diff
@@ -77,9 +77,8 @@ class PARQUET_EXPORT ReaderProperties {
 
 ReaderProperties PARQUET_EXPORT default_reader_properties();
 
-static int64_t DEFAULT_PAGE_SIZE = 1024 * 1024;
+static int64_t DEFAULT_PAGE_SIZE = 64 * 1024 * 1024;
```

Comment:
The default used elsewhere is much smaller. Impala uses 64K, parquet-mr uses 1M. In the absence of benchmarks against different IO sources (e.g. file system vs HDFS) we can probably stick with the 1M default